packaging/aur/test-aur-local.sh

```
@@ -240,15 +321,21 @@ run_variant() {
+        local pkg pkg_count
+        # Exclude -debug- packages and validate exactly one artifact for install.
+        pkg=$(find . -maxdepth 1 -name '*.pkg.tar.*' -type f -printf '%f\n' 2>/dev/null \
+            | grep -v -- '-debug-' | sort | tail -n1)
         if [[ -n "$pkg" ]]; then
             log "[$kind] Build artifact: $pkg"
         fi
 
         if [[ "$kind" = "bin" && "$DO_BUILD" = true && "$DO_INSTALL" = true ]]; then
-            if [[ -z "$pkg" ]]; then
+            pkg_count=$(find . -maxdepth 1 -name '*.pkg.tar.*' -type f -printf '%f\n' 2>/dev/null \
+                | grep -v -- '-debug-' | wc -l)
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)